docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+
+```csharp
+using Microsoft.Maui;
+...
+
+label.SetSemanticFocus();
+```
+
+<!-- Todo: Any scenarios for calling UpdateSemantics or UpdateSemanticNodeInfo? -->
+
+## Semantic screen reader
+
+.NET MAUI Essentials includes a `SemanticScreenReader` class that enables you to instruct a screen reader to announce specified text. <!-- For more information, see [SemanticScreenReader]().-->
+
+<!--
+### Semantic effects
+
+Link to semantic effects (.NET MAUI Community Toolkit) which combines `IsInAccessibleTree` and `ExcludedWithChildren` support, if docs are ever created, + SemanticOrderView.
+-->
+
+## Automation properties
+
+Automation properties are attached properties that can be added to any element to indicate how the element is reported to the underlying platform's accessibility framework.
+
+The `AutomationProperties` class defines the following attached properties:
+
+- `ExcludedWithChildren`, of type `bool?`, determines if an element and its children should be excluded from the accessibility tree. For more information, see [ExcludedWithChildren](#excludedwithchildren).
+- `IsInAccessibleTree`, of type `bool?`, indicates whether the element is available in the accessibility tree. For more information, see [IsInAccessibleTree](#isinaccessibletree).
+- `Name`, of type `string`, represents a short description of the element that serves as a speakable identifier for that element. For more information, see [Name](#name).
+- `HelpText`, of type `string`, represents a longer description of the element, which can be thought of as tooltip text that's associated with the element. For more information, see [HelpText](#helptext).
+- `LabeledBy`, of type `VisualElement`, which enables another element to define accessibility information for the current element. For more information, see [LabeledBy](#labeledby).
+
+These attached properties set native accessibility values so that a screen reader can speak about the element. <!-- For more information about attached properties, see [Attached Properties](~/xamarin-forms/xaml/attached-properties.md). -->
+
+> [!IMPORTANT]
+> Using the `AutomationProperties` attached properties may impact UI Test execution on Android. The `AutomationId`, `AutomationProperties.Name` and `AutomationProperties.HelpText` properties will both set the native `ContentDescription` property, with the `AutomationProperties.Name` and `AutomationProperties.HelpText` property values taking precedence over the `AutomationId` value (if both `AutomationProperties.Name` and `AutomationProperties.HelpText` are set, the values will be concatenated). This means that any tests looking for `AutomationId` will fail if `AutomationProperties.Name` or `AutomationProperties.HelpText` are also set on the element. In this scenario, UI Tests should be altered to look for the value of `AutomationProperties.Name` or `AutomationProperties.HelpText`, or a concatenation of both.
```

Comment:
@PureWeen I can't remember - is this fixed in maui? Is this still an issue, or can Dave remove this warning?

Reply:
This warning is no longer needed. Maybe turn it into a highlight instead? Now with .NET MAUI you can use `AutomationId` and these other properties and everything will stay working!! 

We still need to wire up `HelpText/Name` inside `MAUI` but we're going to do so in such a way that it won't interfere with `AutomationId`. Currently on XF if you use the AccessibilityExperimental_Flag they also don't interfere